src/stubs.ts

```diff
@@ -53,8 +53,154 @@ export module stubs {
         async isBundleRequestAuthorized(request) {
             return true;
         },
-        resourceTypesAllowedForOperation(accessToken, operation): Promise<string[]> {
-            return Promise.resolve([]);
+        getAllowedResourceTypesForOperation(request): string[] {
+            return [

```

Comment:
Could we use R4/R3 constant here instead?

Reply:
There isn't a R4/R3 constant available on this package. I don't think it's worth defining one just for a stub.

Reply:
https://github.com/awslabs/fhir-works-on-aws-interface/blob/mainline/src/constants.ts#L80 ?

Reply:
That's a type. `export type R4Resource`
are you suggesting to change the return type to  `R4Resource[]`? I don't think that's the best since it disallows custom resource types that customers may define.(`AuthorizationRequest.resourceType` is also a string)

Reply:
ooo yeah that is just a `type` yep your right!